frontend/src/components/Navbar.tsx

```diff
@@ -24,6 +24,7 @@ import { Size } from "../utils/size";
 const links = [
   { icon: "home", url: "/", label: "Accueil" },
   { icon: "user", url: "/trombi", label: "Trombi" },
+  { icon: "zap", url: "/repartitions", label: "Repartitions" },
```

Comment:
```suggestion
  { icon: "zap", url: "/repartitions", label: "Répartitions" },
```
Can you find another icon?

Reply:
Maybe `compass` ?